opendistro-for-elasticsearch/plan.sh

```diff
@@ -0,0 +1,95 @@
+ELASTICSEARCH_VERSION="6.5.4"
+ELASTICSEARCH_PKG_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-$ELASTICSEARCH_VERSION.tar.gz"
+ELASTICSEARCH_PKG_SHASUM="0943f434170567e6d780d24caca86d23710049599172a533f8140b2f659dd130"
+pkg_version=0.7.0.1
+pkg_name="opendistro-for-elasticsearch"
+pkg_description="An Apache 2.0-licensed distribution of Elasticsearch enhanced with enterprise security, alerting, SQL, and more"
+pkg_origin="core"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_upstream_url="https://github.com/opendistro-for-elasticsearch"
+pkg_build_deps=(
+  core/git
+  core/maven
+  core/zip
+)
+pkg_deps=(
+  core/busybox-static
```

Comment:
Putting busybox-static first in this list is going to mean that the busybox executables is going to be preferred over those provided by core/coreutils and core/procps-ng in most cases.  If this is what you want, a comment might be good explaining why.

Reply:
@echohack I would recommend making this change before this is merged.  If I followed the thread correctly, this was added to support `free`, which comes from procp-ng which is already a dep, so this dependency may also not be needed.